cards/eye-ddx/card.md

```diff
@@ -0,0 +1,31 @@
+---
+
+title: Rule of 2s for Ocular Emergencies
+author:
+    - Puja Gopal, MD
+created: 2017/04/14
+updates: null
+categories:
+    - Ophthalmology
+ 
+---
+# Background
+
+Eye complaints are a common chief complaint in the emergency department. Having a systematic approach to ocular complaints is especially important as each structure of the eye holds within it different pathology. It is often difficult to remember individual diagnoses for various complaints such as red eye, painful eye, or vision loss, especially during busy shifts. The rule of 2s, as illustrated below, is a simple method with an anatomical approach to the major ocular diagnoses and is meant to provide a general framework to guide the physical exam and include or eliminate certain conditions. The rule of 2s is easy to remember as it contains the number 2 in the context of eyes and proceeds from layer to layer of the eye in an anterior to posterior direction highlighting the 2 major associated diagnoses of that layer.  

```

Comment:
Because these cards are meant to be a quick (potentially bedside) reference, I'd suggest cutting out the "blog"- or "journal"-ish language and get right to the important stuff. (AKA, "trim the fat").

Here's how I would word it (feel free to adjust).

> ~Eye complaints are a common chief complaint in the emergency department. Having a systematic approach to ocular complaints is especially important as each structure of the eye holds within it different pathology. It is often difficult to remember individual diagnoses for various complaints such as red eye, painful eye, or vision loss, especially during busy shifts.~ The rule of 2s, as illustrated below, is a simple method with an anatomical approach to the major ocular diagnoses and is meant to provide a general framework to guide the physical exam and include or eliminate certain conditions. 
>
> The rule of 2s is easy to remember as it contains the number 2 in the context of eyes and proceeds from layer to layer of the eye in an anterior to posterior direction highlighting the 2 major associated diagnoses of that layer.  

Thoughts @jvoros? Am I off base here of would you agree?